charts/mothership/templates/blockscout.yaml

```diff
@@ -44,9 +44,10 @@ spec:
             - name: CHECKSUM_FUNCTION
               value: eth
             - name: DATABASE_URL
-              {{- if .Values.blockscout.db.secretKeyRef }}
+              {{- with .Values.blockscout.db.secretKeyRef }}
```

Comment:
I think `with` cannot be used with `else` below. how about using `db.local` instead?

Reply:
I also thought so, but surprisingly, it seems to work. I tested it on testnet (without secretKeyRef)

Reply:
for reference: https://github.com/golang/go/blob/08d9397170e5870b72a95233e8e4ec43d2d70e30/src/text/template/doc.go#L142